src/services/config-notification.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```

Comment:
Should replace all the 2022 by 2023 (we all forget it :) )

Reply:
done, thank you